pygmt/src/triangulate.py

```diff
@@ -233,7 +233,7 @@ def delaunay_triples(
         ``triangulate`` is a Cartesian or small-geographic area operator and is
         unaware of periodic or polar boundary conditions.
         """
-        output_type = validate_output_table_type(output_type, outfile=outfile)
+        output_type = validate_output_table_type(output_type, outfile)
```

Comment:
Let's keep this change:
```suggestion
        output_type = validate_output_table_type(output_type, outfile=outfile)
```